torchvision/ops/misc.py

```diff
@@ -153,6 +153,6 @@ def forward(self, x):
         b = self.bias.reshape(1, -1, 1, 1)
         rv = self.running_var.reshape(1, -1, 1, 1)
         rm = self.running_mean.reshape(1, -1, 1, 1)
-        scale = w * rv.rsqrt()
+        scale = w / rv.rsqrt()
```

Comment:
Hi,

Just to make sure we are on the same page, the operation we want to do is
```python
scale = w / rv.sqrt()
```
is that right?

This is equivalent to 
```python
scale = w * rv.rsqrt()
```
where `rsqrt(x)` is `1 / sqrt(x)`.

Reply:
You’re right. I missed that.
Thanks!